tests/uuid_ut/uuid_ut.c

```diff
@@ -321,6 +321,43 @@ TEST_FUNCTION(UUID_from_string_succeed)
     }
 }
 
+TEST_FUNCTION(UUID_from_string_invalid_fails)
+{
+    //Arrange
+    int result;
+    UUID_T uuid;
+    const char* invalid_uuids[] = {
```

Comment:
should this be defined at the top of the file?

Reply:
Ok.

Reply:
Done.